public/components/Notifications/NotificationForm.tsx

```diff
@@ -24,8 +25,10 @@ import {
   NotificationChannelOption,
   NotificationChannelTypeOptions,
   TriggerAction,
+  TriggerContext,
 } from '../../../types';
 import { getIsNotificationPluginInstalled } from '../../utils/helpers';
+import Mustache from 'mustache';
```

Comment:
Can we use named imports here and other places too?

Reply:
Done